agatha/construct/semrep_util.py

```diff
@@ -355,6 +383,14 @@ def run(self, input_path:Path, output_path:Path)->None:
     input_path = Path(input_path)
     assert input_path.is_file(), f"Failed to find {input_path}"
     assert not output_path.exists(), f"Refusing to overwrite {output_path}"
+    if line_mode:
```

Comment:
Missing `else`? Right now we run the same thing twice?
I would rather just always use "line mode." It will make things easier to debug if the process does the same thing for every input.

Reply:
Yeah, this function isn't complete yet since I've not figured out how to jump start the `semrep` when this exception occurs, I'm working on that